_research/02-hubs.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Infrastructure for Modeling Hubs
+image: /images/research/modeling-hubs.png
+projects:
+  -  hubverse-org/hubDocs
```

Comment:
@mmkerr can you add some of the other hubverse-org repos here? We could add all of them, but maybe simpler to just add a few key ones, how about:

- hubverse-org/hubverse
- hubverse-org/hubData
- hubverse-org/hubEnsembles